packages/next/edge/iron-session-edge/get-property-descriptor-for-request-session.ts

```diff
@@ -0,0 +1,36 @@
+import type { IronSession } from 'iron-session';
+
+export default function getPropertyDescriptorForRequestSession(
+  session: IronSession
+): PropertyDescriptor {
+  return {
+    enumerable: true,
+    get() {
+      return session;
+    },
+    set(value) {
+      const keys = Object.keys(value);
+      const currentKeys = Object.keys(session);
+
+      for (const key of currentKeys) {
+        if (!keys.includes(key)) {
+          /* eslint-disable @typescript-eslint/no-dynamic-delete */
+          /* eslint-disable @silverhand/fp/no-delete */
```

Comment:
```suggestion
          // eslint-disable-next-line @typescript-eslint/no-dynamic-delete, @silverhand/fp/no-delete
```

Reply:
But the next line is "@ts-expect-error", its effect is also in "next line"

Reply:
ah then you can switch the order i think:

```ts
// @ts-expect-error foo
// eslint-disable-next-line ...
```

Reply:
hhh, unfortunately, `@ts-expect-error` only affects the very next line